manifests/download.pp

```diff
@@ -62,8 +68,12 @@
   Optional[String]                                $proxy_server   = undef,
   Optional[Enum['none', 'http', 'https', 'ftp']]  $proxy_type     = undef,
   Optional[String]                                $url            = undef,
+  Optional[String]                                $username       = undef,
+  Optional[String]                                $password       = undef,
   Boolean                                         $jce            = false,
   Optional[String]                                $jce_url        = undef,
+  Optional[String]                                $jce_username   = undef,
+  Optional[String]                                $jce_password   = undef,
```

Comment:
Need doc strings for these new parameters too.

Reply:
Thanks for checking my PR ! 
THis is now fixed.